### PR TITLE
Update PR review page to remove Travis

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Update PR review page to remove Travis
| Fixed ticket? | Fixes https://github.com/PrestaShop/docs/issues/941

### CODE REVIEW

File: src/content/1.7/contribute/contribution-process/how-pull-requests-are-processed.md [hunk 1]
```diff
@@ -90,6 +78,35 @@ After the Pull Request has finally passed the QA validation, it is merged in the
...
 
+## Other useful resources
+
+- [What it means to rebase a Pull Request submitted on GitHub](https://dev.to/mathieuks/what-it-means-to-rebase-a-pull-request-submitted-on-github-5717) on dev.to, by Mathieu Ferment
+- [How to resolve a git merge conflict](https://opensource.com/article/20/4/git-merge-conflict) on opensource.com by Brian Breniser
+
+---
+
+_(This article was originally published on our blog: [What Happens To Pull Requests After They Are Submitted](https://build.prestashop.com/news/the-review-process/))_
+d to the next step.
```
Comment (NeOMakinG) [inline src/content/1.7/contribute/contribution-process/how-pull-requests-are-processed.md:89]: What do you mean?

**Comment by NeOMakinG** [comment]
Just a question as I didn't really understood, otherwise it seems good

File: src/content/1.7/contribute/contribution-process/how-pull-requests-are-processed.md [hunk 2]
```diff
@@ -13,30 +13,18 @@ When you submit a new Pull Request to the [project repository](https://github.co
 
 ### Continuous Integration
 
-Four automatic code checker tools are active on the project.
-
 #### GitHub Actions
```
Comment (eternoendless) [inline src/content/1.7/contribute/contribution-process/how-pull-requests-are-processed.md:17]: You can remove this title

```suggestion
```

File: src/content/1.7/contribute/contribution-process/how-pull-requests-are-processed.md [hunk 3]
```diff
@@ -13,30 +13,18 @@ When you submit a new Pull Request to the [project repository](https://github.co
...
-
-If the Github Action states that your pull request has issues, you need to fix the issues by looking at the Github Action output to understand what needs to be fixed.
-
-The second tool is a [ESLint](https://eslint.org/) GitHub Action. This one checks whether the Javascript syntax and code-style is correct. Likewise, it will block the PR from being merged if there is a syntax or code-style issue.
-
-The third tool is a [stylelint](https://stylelint.io/) GitHub Action. This one checks whether the CSS syntax and code-style is correct. It will block the PR from being merged as well if there is a syntax or code-style issue.
+The project relies on [GitHub Action](https://github.com/features/actions) to perform multiple checks against submitted Pull Requests.
 
-These GitHub Actions are executed automatically for every Pull Request.
+Checks being performed include:
```
Comment (eternoendless) [inline src/content/1.7/contribute/contribution-process/how-pull-requests-are-processed.md:20]:
```suggestion
These checks include:
```

File: src/content/1.7/contribute/contribution-process/how-pull-requests-are-processed.md (hunk 1 above)
Comment (eternoendless) [inline src/content/1.7/contribute/contribution-process/how-pull-requests-are-processed.md:89]:
```suggestion
```

File: src/content/1.7/contribute/contribution-process/how-pull-requests-are-processed.md [hunk 4]
```diff
@@ -90,6 +78,35 @@ After the Pull Request has finally passed the QA validation, it is merged in the
...
+
+After the Pull Request has finally passed the QA validation, it is merged in the project and the author becomes (if they weren't already) a contributor to this great open source project !
+
+### Graphical summary
+
+(click on it to see full size)
+
+![Pull Request process summary](../../img/pull-request-process.png)
+
+
```
Comment (eternoendless) [inline src/content/1.7/contribute/contribution-process/how-pull-requests-are-processed.md:109]: Duplicate content
```suggestion
```

**Review by eternoendless — CHANGES_REQUESTED** [review]
Some changes required

File: src/content/1.7/contribute/contribution-process/how-pull-requests-are-processed.md (hunk 1 above)
Comment (matks) [reply src/content/1.7/contribute/contribution-process/how-pull-requests-are-processed.md:89]: The content of this page was initially published on https://build.prestashop.com/news/the-review-process/ then moved into the doc. This message informs the user about this history.

Comment (matks) [reply src/content/1.7/contribute/contribution-process/how-pull-requests-are-processed.md:89]: Oh! I understand now. My PR is messed up.

File: src/content/1.7/contribute/contribution-process/how-pull-requests-are-processed.md (hunk 4 above)
Comment (matks) [reply src/content/1.7/contribute/contribution-process/how-pull-requests-are-processed.md]: Fixed https://github.com/PrestaShop/docs/pull/946/commits/f22f75fe608e352a959e06a1e69bac823f603256

File: src/content/1.7/contribute/contribution-process/how-pull-requests-are-processed.md (hunk 1 above)
Comment (matks) [reply src/content/1.7/contribute/contribution-process/how-pull-requests-are-processed.md]: Fixed https://github.com/PrestaShop/docs/pull/946/commits/f22f75fe608e352a959e06a1e69bac823f603256

**Review by eternoendless — APPROVED** [review]

**Comment by eternoendless** [comment]
Thank you @matks

**Pull request merged**